Index: trend-lab-service/src/main/java/com/stackroute/trendlabservice/model/Model.java
===================================================================
diff --git a/trend-lab-service/src/main/java/com/stackroute/trendlabservice/model/Model.java b/trend-lab-service/src/main/java/com/stackroute/trendlabservice/model/Model.java
deleted file mode 100644
--- a/trend-lab-service/src/main/java/com/stackroute/trendlabservice/model/Model.java	(revision f446a9913b9bf54fa88016b49d1f98b8598ac80c)
+++ /dev/null	(revision f446a9913b9bf54fa88016b49d1f98b8598ac80c)
@@ -1,4 +0,0 @@
-package com.stackroute.trendlabservice.model;
-
-public class Model {
-}
